disttask/importinto/dispatcher.go

```diff
@@ -223,18 +226,28 @@ func (dsp *ImportDispatcherExt) OnNextSubtasksBatch(ctx context.Context, taskHan
 		}
 	}()
 
+	var previousSubtaskMetas [][]byte
 	switch gTask.Step {
 	case proto.StepInit:
 		if metrics, ok := metric.GetCommonMetric(ctx); ok {
 			metrics.BytesCounter.WithLabelValues(metric.StateTotalRestore).Add(float64(taskMeta.Plan.TotalFileSize))
 		}
-		if err := preProcess(ctx, taskHandle, gTask, taskMeta, logger); err != nil {
+		jobStep := importer.JobStepImporting
```

Comment:
and these logic seems more natural as a part of `GetNextStep`

Reply:
no, that's `import-into` job's step, exposed to user and has less steps